Makefile.am

```diff
@@ -19,7 +19,7 @@ include ./branch-config.mk
 
 ACLOCAL_AMFLAGS = -I m4
 
-SUBDIRS = data docs dracut po pyanaconda scripts tests widgets ui utils
```

Comment:
I think it makes more sense, to backport your --disable-webui commit, instead of doing this hack.

Removing just the subdirs is really weird way to ignore the existing code.

Reply:
If we backport that solution, we have to enable that everywhere which seems a bit unnecessary work honestly. Again we don't want to use the code here ever.